falcon_multiqc/commands/sql.py

```diff
@@ -0,0 +1,52 @@
+import click
+import glob
+import os
+from sqlalchemy import create_engine
+from database.config import DATABASE_URI
+from database.process_query import run_multiqc
+
+# example query: 
+
+# this query will query fastqc tool for Total seqence metric - since only the first batch ran fastqc it will only return first batch
+# SELECT s.sample_name, b.path FROM public.raw_data as rd join sample as s on (s.id = rd.sample_id) join batch as b on (s.batch_id = b.id) where rd.qc_tool = 'fastqc' AND (rd.metrics->>'Total Sequences')::numeric::integer >= 316624794;
+
+# this queries two different tools (picard_insertsize and verifybamid) and on of their respective metrics - returns sampleID from first batch and sampleID from 4012 batch 
+# SELECT s.sample_name, b.path FROM public.raw_data as rd join sample as s on (s.id = rd.sample_id) join batch as b on (s.batch_id = b.id) where ((rd.qc_tool = 'picard_insertSize' AND (rd.metrics->>'MEAN_INSERT_SIZE')::numeric::integer <= 390) OR (rd.qc_tool = 'verifybamid' AND (rd.metrics->>'#READS')::numeric::integer >= 4850000));
+
+@click.command()
+@click.option("-d", "--directory", type=click.STRING, required=True, help="where query result will be saved")
+@click.option("-s", "--sql", type=click.INT, required=False, help="Enter 1 to start SQL querying tool") 
```

Comment:
Hii,  awesome, I have a few comments! 

I don't see why we have this option? I think maybe change this to take in a string with the actual SQL. Then in the method we check if this is provided, if not we straight away ask for the SQL (and no "if sql ==1" etc.)

Reply:
Hey yeah the only reason I gave that option was because I was thinking of how this could integrate with the query command.  I could get rid of it if we make the sql command a stand alone feature? - anyway I changed it to accept the full sql statement instead for now haha